contracts/DecentralizedAutonomousTrust.sol

```diff
@@ -13,13 +13,17 @@ import "@openzeppelin/contracts-ethereum-package/contracts/utils/Address.sol";
 
 /**
  * @title Decentralized Autonomous Trust
- * This contract is the reference implementation provided by Fairmint for a
+ * This contract is the copy of the implementation provided by Fairmint for a
```

Comment:
Should we say "modified version" since we made edits?

Reply:
yep